### DISCUSSION TITLE
PR Excluded in hacktoberfest2024

### DISCUSSION THREAD

**Discussion by abishek1740P** [DevAffan/AffiCodes#226] [General]
Why is my PR raised in this repo are getting excluded in the Hacktoberfest2024.
@DevAffan 
![Screenshot 2024-10-02 172138](https://github.com/user-attachments/assets/d5ba5495-97e0-4c1b-9ee5-d9915a08a066)

**Comment by DevAffan** [comment]
Thank you for letting me know. Can you please hover on the "?" icon and check the reason for the exclusion?

**Comment by abishek1740P** [comment]
![image](https://github.com/user-attachments/assets/e063d4a4-96e7-4255-8824-8f115382c201)
@DevAffan

**Comment by DevAffan** [comment]
Repository Exclusion from Hacktoberfest – Apologies and Clarification

Hello everyone,

I wanted to take a moment to address the recent exclusion of my repository from Hacktoberfest. It has come to my attention that my repository was reported or excluded by Hacktoberfest due to a high volume of pull requests related to Data Structures and Algorithms (DSA), some of which may have included copied content.

First and foremost, I want to sincerely apologize for any confusion or inconvenience this has caused. Please know that this was not my intention, and I did not anticipate such an outcome. The purpose of this repository was to provide a meaningful contribution platform for everyone, and I fully support original, high-quality contributions.

I am working to resolve this issue and will take steps to prevent similar occurrences in the future.

Thank you for your understanding and continued support.

Best regards,
Affan Ali
